alarm/utils.py

```diff
@@ -30,6 +37,33 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
                     f"threshold: {threshold}; "
                     f"threshold broken: {threshold_broken}")
         if threshold_broken:
+            ThresholdBrake.objects.create(threshold=threshold)
             return True
 
     return False
+
+
+def check_call_status(call, message):
+    if call.status == 'completed':
+        # Sleep for 15 minutes
+        time.sleep(900)
+    elif call.status == 'busy' or call.status == 'failed':
+        # Sleep for 1 minute and create a new call
+        time.sleep(60)
+        call = twilio_client.calls.create(
+            twiml=message,
+            to=USER_PHONE_NUMBER,
+            from_=PHONE_NUMBER_TWILLIO
+        )
+        check_call_status(call, message)
+    else:
```

Comment:
Twillio status call:
queued: The call is waiting in a queue before being processed.
ringing: The call is currently ringing and waiting for someone to answer.
in-progress: The call has been answered and is currently in progress.
canceled: The call was canceled before it could be answered.
completed: The call has completed and ended.
busy: The call could not be completed because the line was busy.
failed: The call failed for some reason (e.g. network issues).
no-answer: The call was not answered by the recipient.
answered: The call was answered, but it is not yet known whether it is in progress or completed.